pyaedt/modules/Material.py

```diff
@@ -369,8 +351,8 @@ def add_thermal_modifier_free_form(self, formula, index=0):
 
         Returns
         -------
-        bool
-            ``True`` when successful, ``False`` when failed.
+        type
```

Comment:
```suggestion
        bool
            ``True`` when successful, ``False`` when failed.
```
The call to `_add_thermal_modifier` ultimately calls `update` on the `Material` class, which returns true or false.